packages/react/src/components/AccountSettings/ChangePassword/types.ts

```diff
@@ -1,6 +1,10 @@
+import { FieldValidator } from '@aws-amplify/ui';
+
 export interface ChangePasswordProps {
   // callback once password is successfully updated
   onSuccess?: () => void;
   // callback when there's an error
   onError?: (error: Error) => void;
+  // custom password validations
```

Comment:
This is a scenario where i think JSDoc syntax is helpful, just noting

Reply:
Converted to jsdoc comment in [9a9fa07](https://github.com/aws-amplify/amplify-ui/pull/2900/commits/9a9fa073a640b861a26ab719782a0d43fb78a6e3)